packages/tools/sandbox/src/components/renderingZone.tsx

```diff
@@ -318,6 +319,11 @@ export class RenderingZone extends React.Component<IRenderingZoneProps> {
             this._engine.runRenderLoop(() => {
                 // Adapt the camera sensibility based on the distance to the object
                 camera.panningSensibility = 5000 / camera.radius;
+
+                // Ensure the panning epsilon is always small enough (no bigger than a 10th of the smallest amount
+                // of movement the user can cause) so that even when the panningSensibility is very high and the
+                // user pans very slowly, we can still get inertia during the pan without it being clamped to 0 too quickly.
+                camera._panningEpsilon = Math.min(Epsilon, 1 / camera.panningSensibility / 10);

```

Comment:
Maybe `_panningEpsilon ` shouldn't be an underscore variable or maybe it should also update when `panningSensibility` is updated.